docs/griptape-framework/structures/pipelines.md

```diff
@@ -41,7 +41,7 @@ pipeline.run("Write me a haiku about sailing.")
                              Sea whispers secrets to hull,
                              Horizon awaits.
 [09/08/23 10:19:21] INFO     PromptTask 28e36610063e4d728228a814b48296ef
-                             Output: Yarr! Th' sails snag th' mornin' zephyr,
+                             Output: Yarr! Th' sails snag th' morning zephyr,
```

Comment:
is there a way to give it pirate syntax mode?

Reply:
Pirate mode engaged